frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -46,10 +45,17 @@ void main(List<String> args) async {
     expect(resp.int16List, Int16List.fromList(List.filled(n, 42)));
     expect(resp.uint32List, Uint32List.fromList(List.filled(n, 42)));
     expect(resp.int32List, Int32List.fromList(List.filled(n, 42)));
-    expect(resp.uint64List, Uint64List.fromList(List.filled(n, 42)));
-    expect(resp.int64List, Int64List.fromList(List.filled(n, 42)));
     expect(resp.float32List, Float32List.fromList(List.filled(n, 42)));
     expect(resp.float64List, Float64List.fromList(List.filled(n, 42)));
+    try {
+      expect(resp.uint64List, Uint64List.fromList(List.filled(n, 42)));
+      expect(resp.int64List, Int64List.fromList(List.filled(n, 42)));
+    } on TestFailure {
+      rethrow;
+    } catch (err, st) {
+      print('$err\n$st');
+      markTestSkipped('Skip: Matcher has problems with JS native types');
```

Comment:
nit: dart test seems to support platform-specific configurations

e.g. https://pub.dev/packages/test#restricting-tests-to-certain-platforms

Reply:
I'll remove these once I get them to work.

Reply:
Oh I see. Take your time :)

Btw, though have said a thousand time, I still want to say this is a great PR!